src/test/java/core/basesyntax/service/impl/ReportMakerImplTest.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitCrate;
+import core.basesyntax.service.ReportMaker;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ReportMakerImplTest {
+    private static ReportMaker reportMaker;
+    private static List<FruitCrate> batch;
+    private static String report;
+
+    @BeforeClass
+    public static void beforeClass() {
+        reportMaker = new ReportMakerImpl();
+        report = "fruit,quantity" + System.lineSeparator()
+                + "apple,63" + System.lineSeparator()
+                + "banana,125";
+        batch = new ArrayList<>(List.of(new FruitCrate("apple", 63),
+                new FruitCrate("banana", 125)));
```

Comment:
```suggestion
        batch = List.of(new FruitCrate("apple", 63),
                new FruitCrate("banana", 125));
```

Reply:
Then I will get UnsupportedOperationException in future